src/components/Inventory/index.tsx

```diff
@@ -81,61 +85,93 @@ export default function Inventory(props: InventoryProps): JSX.Element {
 
   const isOnboarded = hasNurseries && hasSpecies;
 
-  useEffect(() => {
-    let activeRequests = true;
-
-    if (organization) {
-      const populateResults = async () => {
-        const apiResponse = await search({
-          prefix: 'inventories',
-          fields: [
-            'species_id',
-            'species_scientificName',
-            'species_commonName',
-            'facilityInventories.facility_name',
-            'germinatingQuantity',
-            'notReadyQuantity',
-            'readyQuantity',
-            'totalQuantity',
-          ],
-          search: {
-            operation: 'and',
-            children: [
-              {
-                operation: 'or',
-                children: [
-                  { operation: 'field', field: 'species_scientificName', type: 'Fuzzy', values: [debouncedSearchTerm] },
-                  { operation: 'field', field: 'species_commonName', type: 'Fuzzy', values: [debouncedSearchTerm] },
-                  {
-                    operation: 'field',
-                    field: 'facilityInventories.facility_name',
-                    type: 'Fuzzy',
-                    values: [debouncedSearchTerm],
-                  },
-                ],
-              },
-              {
-                operation: 'field',
-                field: 'organization_id',
-                values: [organization.id],
-              },
-            ],
+  const getParams = useCallback(() => {
```

Comment:
wondering if all this is better pushed into the table component...
we would need some way to identify the empty state though.

Reply:
Yes, I tried to do that but I was repeating code to see if the inventory has columns or not